cmd/gitService.go

```diff
@@ -0,0 +1,119 @@
+package cmd
+
+import (
+	_ "embed"
+	"os"
+	"os/exec"
+
+	log "github.com/sirupsen/logrus"
+)
+
+var DEPLOY_CONFIG_GIT_REPO string = "https://automotivemastermind@dev.azure.com/automotivemastermind/aM/_git/am.devops.deploy"
+var DEPLOY_CONFIG_GIT_REPO_BRANCH string = "local"
+
+var HELM_CONFIG_GIT_REPO string = "https://automotivemastermind@dev.azure.com/automotivemastermind/aM/_git/am.devops.helm"
+var HELM_CONFIG_GIT_REPO_BRANCH string = "local"
+
+//var FPS string = string(filepath.Separator)
```

Comment:
who are you and why are you commented out!

Reply:
if your looking to make this work with windows you probably can use the [path/filepath](https://pkg.go.dev/path/filepath) package to create file uri's.

for web Uris just use the path package

Reply:
filepath.Separator comes from path/filepath package. We get the native separator and use it throughout the application.